### DISCUSSION TITLE
Can I rename renovate.json to ".renovate.json"? does renovatebot will be able to process it?

### DISCUSSION THREAD

**Discussion by bhargu99** [renovatebot/tutorial#40] [Q&A] [answered]
As it is config file, I wanted to replace the renovate.json to ".renovate.json". will renovate bot able to understand that?

I saw from configuration it will be able to understand only below formats.

renovate.json
renovate.json5
.github/renovate.json
.github/renovate.json5
.gitlab/renovate.json
.gitlab/renovate.json5
.renovaterc
.renovaterc.json
.renovaterc.json5
package.json (within a "renovate" section)

**Comment by rarkins** [comment, marked as answer]
No
